.circleci/config.yml

```diff
@@ -184,6 +184,7 @@ build_and_deploy: &build_and_deploy
           # Force add all files to include those excluded by .gitignore
           # Before force-adding, remove large folders we don't need on Pantheon
           cd /home/circleci/app && find ./web/modules -name .git -type d -prune -exec rm -rf "{}" \;
+          find ./web/libraries -name .git -type d -prune -exec rm -rf "{}" \;
```

Comment:
When the third party library was loaded from GitHub by Composer, the ".git" folder prevented the library from being pushed to Pantheon. Like the line above, removing these ".git" folders appears to help.

Reply:
Ah, I see. It's kind of odd how the first one changes directory so that the relative `./web/modules` path is correct. Your new line uses the same relative path structure so it is also dependent on the previous line's `cd /home/circleci/app`... I think it would be better if both lines just used an absolute path like `/home/circleci/app/web/whatever`. That's simpler than having to change directories and then if we ever remove the first line for any reason, the second line will still work.